### PR TITLE
Version 1.0.0

### PR DESCRIPTION
**Pull request by CoderTCY**

### CODE REVIEW

**Comment by shubhodeep9** [comment]
Hi @TanChenyu 
is this PR complete and ready to review?

File: setup.py [hunk 1]
```diff
@@ -19,22 +18,16 @@
...
-
 # Import the README and use it as the long-description.
 # This will only work if 'README.rst' is present in your MANIFEST.in file!
-with io.open(os.path.join(here, 'README.md'), encoding='utf-8') as f:
+with open('README.md', encoding='utf-8') as f:
     long_description = '\n' + f.read()
 
-# Load the package's __version__.py module as a dictionary.
-about = {}
-with open(os.path.join(here, NAME, '__version__.py')) as f:
```
Comment (shubhodeep9) [inline setup.py:31]: any specific reason you removed the separate version file?

File: pipsearch/api.py [hunk 2]
```diff
@@ -3,47 +3,53 @@
...
+    pn = int(soup.find_all(class_="button button-group__button")[-2].text)
+    get_packages(soup)
 
-    # Constructing the result list
-    packages = []
+    # Constructing a search URL and sending the request
+    # Concatinating the results from splitting will lead to good results
+    for i in range(2, pn+1):
+        soup = BeautifulSoup(requests.get(
+                "https://pypi.org/search",
```
Comment (shubhodeep9) [inline pipsearch/api.py:48]: Can we move the URI to a constant which we can call where we require?

**Review by shubhodeep9 — CHANGES_REQUESTED** [review]
Also, it helps if you summarise what changes have been made in the description

File: setup.py (hunk 1 above)
Comment (CoderTCY) [reply setup.py:31]: Because I think it is unnessary.

File: pipsearch/api.py (hunk 2 above)
Comment (CoderTCY) [reply pipsearch/api.py:48]: Yes.

**Comment by CoderTCY** [comment]
> 
> 
> Hi @TanChenyu
> is this PR complete and ready to review?

Maybe. I am planning to create a new PR to instead it.

**Pull request closed without merging**